### PR TITLE
#5625 Bug fix - Fix quick bar shortcut display in the activate side panel UI

### LINKED ISSUE #5625: Bug - Quick Bar activation success info shortcut UI is broken on Windows machines
Context: https://pixiebrix.slack.com/archives/C02CN01JXAA/p1682619653723699

On Windows, the Quick Bar shortcut (string) coming back from the chrome api is `Ctrl+M`, where on MacOS it comes back as just `⌘M`:

image

### PR DESCRIPTION
**Pull request by BLoe**
## What does this PR do?

- Fixes #5625 

## Demo

image

## Checklist

- [ ] Add tests
- [ ] Designate a primary reviewer

### CODE REVIEW

File: src/sidebar/activateRecipe/ActivateRecipePanel.tsx [hunk 1]
```diff
@@ -46,13 +46,14 @@ import { persistor } from "@/sidebar/store";
 const { actions } = sidebarSlice;
 
 const ShortcutKeys: React.FC<{ shortcut: string | null }> = ({ shortcut }) => {
-  const shortcutKeys = shortcut?.split("") ?? [];
+  const separator = shortcut?.includes("+") ? "+" : "";
```
Comment (twschiller) [inline src/sidebar/activateRecipe/ActivateRecipePanel.tsx:49]: @BLoe if someone did "Ctrl" as the shortcut, this would show ["C", "t", "r", "l"], corect?

Comment (twschiller) [reply src/sidebar/activateRecipe/ActivateRecipePanel.tsx:49]: You could consider doing OS detection here as well?

**Comment by twschiller** [comment]
@BLoe looks like snapshots/tests need to be updated

File: src/sidebar/activateRecipe/ActivateRecipePanel.tsx (hunk 1 above)
Comment (BLoe) [reply src/sidebar/activateRecipe/ActivateRecipePanel.tsx:49]: I checked using the rainforest VM how the chrome shortcuts work on Windows. The shortcut must be either Ctrl or Alt and then a "letter" key ("letter" can be a period or some other characters too). The shortcut is returned from the chrome API as `Ctrl+M` or `Alt+M`. On MacOS, they use the icons and you also can use Option. And the shortcuts don't have a + on MacOS, it's just `⌘M` or `⌥M`.

Comment (mnholtz) [reply src/sidebar/activateRecipe/ActivateRecipePanel.tsx:49]: Why do we need to apply `split()` at all to shortcuts that don't have a `+`?

**Review by mnholtz — APPROVED** [review]
Approved, but see my question here: https://github.com/pixiebrix/pixiebrix-extension/pull/5626/files#r1180667039

**Comment by BLoe** [comment]
Passing rainforest test: https://app.rainforestqa.com/runs/1315603

File: src/sidebar/activateRecipe/ActivateRecipePanel.tsx (hunk 1 above)
Comment (BLoe) [reply src/sidebar/activateRecipe/ActivateRecipePanel.tsx:49]: The + is returned on Windows but not MacOS

**Pull request merged**